### PR TITLE
[Correction] Nepali Language, meridiem and days of week.

### PR DESCRIPTION
**Pull request by njsubedi**
Fixed-
1. Grammatically corrected the words
2. Corrected the mini-form of week days
2. Corrected the meridiem time
3. Corrected day of week and day of year

### CODE REVIEW

**Comment by ichernev** [comment]
@suvash what do you think?

**Comment by ichernev** [comment]
@njsubedi you have to make changes to `src/locale/ne.js`.
Also you have to modify the corresponding tests (so they pass) in `src/test/locale/ne.js`.

Run tests with `grunt test` (you need npm / node set up)

File: locale/ne.js [hunk 1]
```diff
@@ -39,7 +39,7 @@
         monthsShort : 'जन._फेब्रु._मार्च_अप्रि._मई_जुन_जुलाई._अग._सेप्ट._अक्टो._नोभे._डिसे.'.split('_'),
         weekdays : 'आइतबार_सोमबार_मङ्गलबार_बुधबार_बिहिबार_शुक्रबार_शनिबार'.split('_'),
         weekdaysShort : 'आइत._सोम._मङ्गल._बुध._बिहि._शुक्र._शनि.'.split('_'),
-        weekdaysMin : 'आइ._सो._मङ्_बु._बि._शु._श.'.split('_'),
+        weekdaysMin : 'आ._सो._मं_बु._बि._शु._श.'.split('_'),
```
Comment (suvash) [inline locale/ne.js]: Seem like i have missed the (.) previously. Maybe you could add it back (on Tuesday/मंगलबार) ->  मं.

File: locale/ne.js [hunk 2]
```diff
@@ -58,39 +58,39 @@
                 return symbolMap[match];
             });
         },
-        meridiemParse: /राती|बिहान|दिउँसो|बेलुका|साँझ|राती/,
+        // We only have four state of time.
+        // साँझ and बेलुका are synonyms only!
+        meridiemParse: /राति|बिहान|दिउँसो|साँझ/,
```
Comment (suvash) [inline locale/ne.js]: I could agree to this, if not being very pedantic. The comments could have been placed in the discussion itself. Maybe remove them from here.

File: locale/ne.js [hunk 3]
```diff
@@ -112,11 +112,11 @@
             yy : '%d बर्ष'
         },
         week : {
-            dow : 1, // Monday is the first day of the week.
-            doy : 7  // The week that contains Jan 1st is the first week of the year.
+            dow : 1, // Sunday is the first day of the week in Nepal
+            doy : 1 // The week that contains Jan 1st is the first week of the year.
```
Comment (suvash) [inline locale/ne.js]: I cannot remember what this was supposed to represent(and why it has been changed). It's been a while.

**Comment by suvash** [comment]
@njsubedi As @ichernev has mentioned you will have to update the tests as well.

File: locale/ne.js (hunk 3 above)
Comment (ichernev) [inline locale/ne.js]: If sunday is the first day of the week, and the first week is the one containing 1st Jan, then the numbers should be `dow: 0, doy: 6`. 1, 1 is definitely not correct.

**Comment by ichernev** [comment]
@suvash, thank you!

**Comment by ichernev** [comment]
@njsubedi can you please fix the comments. You also need to change files under `src/locale` and update tests under `src/test/locale`

**Comment by ichernev** [comment]
Close in favor of #2738

**Pull request closed without merging**